plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/IRouteChooser.java

```diff
@@ -20,6 +20,17 @@
 public interface IRouteChooser {
 
 	public IRoute chooseRoute(List<IRoute> routes);
+
+	/**
+	 * Returns user's decision to remember the chosen route.
+	 * This method should be called after chooseRoute(List<IRoute).
+	 * 
+	 * @return
+	 */
+	default public boolean isRememberChoice() {
```

Comment:
That is true, but I only added one new method to an interface that already had methods with 'public' modifier. So I had to keep to the style. It will be good to have either all with modifier or none. A mix is not nice. Will you recommend removing 'public' at the other methods?